src/routes/EventsRoute.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import Header from "../components/modules/Header";
+import PageBody from "../components/common/PageBody";
+import Footer from "../components/modules/Footer";
+
+export default class EventsRoute extends React.Component {
+    constructor() {
+        super();
```

Comment:
If the only thing in the `constructor` is a call to `super()`, you actually just omit the constructor entirely and it will be done automatically!

Reply:
Thanks! I've omitted the constructor.